### PR TITLE
Add modules api

### PR DESCRIPTION
**Pull request by juliangruber**
cc @davelange

### CODE REVIEW

File: renderer/src/hooks/StationModules.tsx [hunk 1]
```diff
@@ -1,24 +1,74 @@
...
 
-type Module = typeof mockData[number]
+/* eslint-disable max-len */
+const modules = {
+  spark: {
+    name: 'Spark',
+    description: 'With its array of tools, resources, and support, Spark empowers users to spark new ideas, projects, and solutions that harness the full potential of Filecoin\'s decentralized storage protocol.',
+    contractAddress: '0x',
+    logo: 'https://assets-global.website-files.com/633bfdb71d8e552b70ceff10/633eb29c75502c57fa5ff646_station-download.svg',
+    stats: { // TODO: Get this from spark-stats once available
```
Comment (juliangruber) [inline renderer/src/hooks/StationModules.tsx:26]: this data isn't available yet anywhere

File: renderer/src/hooks/StationModules.tsx [hunk 2]
```diff
@@ -1,24 +1,74 @@
...
 }
 
-type Module = typeof mockData[number]
+/* eslint-disable max-len */
+const modules = {
+  spark: {
+    name: 'Spark',
+    description: 'With its array of tools, resources, and support, Spark empowers users to spark new ideas, projects, and solutions that harness the full potential of Filecoin\'s decentralized storage protocol.',
+    contractAddress: '0x',
+    logo: 'https://assets-global.website-files.com/633bfdb71d8e552b70ceff10/633eb29c75502c57fa5ff646_station-download.svg',
```
Comment (juliangruber) [inline renderer/src/hooks/StationModules.tsx:25]: can we inline this svg? Do you think it's a good idea for a hook to import and expose an svg?

File: renderer/src/hooks/StationModules.tsx [hunk 3]
```diff
@@ -1,24 +1,74 @@
...
+      totalRewards: 0
+    },
+    links: {
+      github: 'https://github.com/filecoin-station/spark',
+      docs: 'https://filspark.com/'
+    }
+  },
+  voyager: {
+    name: 'Voyager',
+    description: 'Module Y empowers users to spark new ideas, projects, and solutions that harness the full potential of Filecoin\'s decentralized storage protocol.',
```
Comment (juliangruber) [inline renderer/src/hooks/StationModules.tsx:37]: @patrickwoodhead do we have a quick description for the Voyager module yet?

File: renderer/src/hooks/StationModules.tsx [hunk 4]
```diff
@@ -1,24 +1,74 @@
...
+    contractAddress: '0xc524b83bf85021e674a7c9f18f5381179fabaf6c',
+    logo: 'https://assets-global.website-files.com/633bfdb71d8e552b70ceff10/633eb29c75502c0e935ff647_station-connect.svg',
```
Comment (juliangruber) [inline renderer/src/hooks/StationModules.tsx:39]: same here as above

File: renderer/src/hooks/StationModules.tsx [hunk 5]
```diff
@@ -1,24 +1,74 @@
...
+    stats: {
```
Comment (juliangruber) [inline renderer/src/hooks/StationModules.tsx:40]: this data isn't available yet

File: renderer/src/hooks/StationModules.tsx (hunk 2 above)
Comment (davelange) [reply renderer/src/hooks/StationModules.tsx:25]: Yes, we could inline them 👍

**Review by davelange — APPROVED** [review]

File: renderer/src/hooks/StationModules.tsx (hunk 4 above)
Comment (juliangruber) [reply renderer/src/hooks/StationModules.tsx:40]: TODO: Get a Voyager logo

**Comment by juliangruber** [comment]
I'm going to create a follow-up PR once we have more module information

**Pull request merged**